src/ui/components/Video.tsx

```diff
@@ -75,7 +75,7 @@ function Video({
     if (playback) {
       refreshGraphics();
       VideoPlayer.seek(currentTime);
-      VideoPlayer.play();
+      VideoPlayer.play(currentTime);
```

Comment:
Is this value the same as what `protocol/graphics` was using before?

https://github.com/RecordReplay/devtools/blob/ae52cb0d709e2a1b480697d8afa9243b05c60f8a/packages/protocol/graphics.ts#L232